test/number/bitwise_precedence.wren

```diff
@@ -0,0 +1,13 @@
+IO.print(2 | 1 << 1) // expect: 2
```

Comment:
Yay tests! This test really isn't about numbers as much as it is syntax (though it, of course, uses method calls on numbers to verify the parse), so let's move it up to just `test/` next to `precedence.wren`.